package.json

```diff
@@ -95,7 +95,6 @@
     "protractor-jasmine2-screenshot-reporter": "^0.5.0",
     "puppeteer": "^13.0.0",
     "reflect-metadata": "^0.1.12",
-    "rtlcss": "^3.5.0",
```

Comment:
Hmm, why is this removed?

Reply:
Hmm, this was automatic and after searching for rtlcss I did not find any instance in the codebase. `rtlcss` is added in https://github.com/oppia/oppia/pull/14375 which is not a part of the release.

Reply:
@aks681 can you confirm what should be done here? Thanks!

Reply:
Cool, thanks!